src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -303,28 +303,33 @@ public Compilation Clone()
         /// <param name="ignoreAccessibility">
         /// True if the SemanticModel should ignore accessibility rules when answering semantic questions.
         /// </param>
+#pragma warning disable RS0027 // API with optional parameter(s) should have the most parameters amongst its public overloads
```

Comment:
I assumed that suppression of this is the best choice while the SemanticModelOptions overload is still experimental. For existing calls which were just passing `syntaxTree`, we want the non-experimental method to be chosen.